site-manage-group-section-role-helper/tool/src/java/org/sakaiproject/site/tool/helper/managegroupsectionrole/rsf/JoinableGroupDelProducer.java

```diff
@@ -0,0 +1,98 @@
+package org.sakaiproject.site.tool.helper.managegroupsectionrole.rsf;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.sakaiproject.site.api.Group;
+import org.sakaiproject.site.tool.helper.managegroupsectionrole.impl.SiteManageGroupSectionRoleHandler;
+import uk.ac.cam.caret.sakai.rsf.producers.FrameAdjustingProducer;
+import uk.org.ponder.messageutil.MessageLocator;
+import uk.org.ponder.messageutil.TargettedMessageList;
+import uk.org.ponder.rsf.components.*;
+import uk.org.ponder.rsf.components.decorators.UILabelTargetDecorator;
+import uk.org.ponder.rsf.flow.ARIResult;
+import uk.org.ponder.rsf.flow.ActionResultInterceptor;
+import uk.org.ponder.rsf.view.ComponentChecker;
+import uk.org.ponder.rsf.view.ViewComponentProducer;
+import uk.org.ponder.rsf.viewstate.SimpleViewParameters;
+import uk.org.ponder.rsf.viewstate.ViewParameters;
+import uk.org.ponder.rsf.viewstate.ViewParamsReporter;
+
+/**
+ * 
+ * @author
+ *
+ */
+public class JoinableGroupDelProducer
+implements ViewComponentProducer, ActionResultInterceptor, ViewParamsReporter {
+
+	/** Our log (commons). */
+	private static Log M_log = LogFactory.getLog(JoinableGroupDelProducer.class);
+	
+	public static final String VIEW_ID = "JoinableGroupDel";
+	public MessageLocator messageLocator;
+	public SiteManageGroupSectionRoleHandler handler;
+	public FrameAdjustingProducer frameAdjustingProducer;
```

Comment:
You don't seem to be using this. If you want the frame to automagically adjust you need to added it to the components.